src/main/java/com/criteo/nosql/casspoke/cassandra/CassandraMetrics.java

```diff
@@ -14,14 +14,14 @@
     public static final Gauge UP = Gauge.build()
             .name("cassandra_up")
             .help("Are the servers up?")
-            .labelNames("cluster", "instance", "datacenter")
+            .labelNames("cassandra_cluster", "cassandra_probe_location", "cassandra_instance", "cassandra_datacenter")
```

Comment:
Can you make the prefix configurable? Like `cassandraPrefix = ""` per default and `cassandraPrefix = "cassandra_" `in your configuration. It's to avoid a breaking change on the metrics labels.